server/routerlicious/packages/memory-orderer/package.json

```diff
@@ -39,7 +39,7 @@
     "@types/node": "^10.17.24",
     "@types/ws": "^6.0.1",
     "debug": "^4.1.1",
-    "double-ended-queue": "^2.1.0-0",
```

Comment:
i don't think this is our code

Reply:
you should only be updated our packages specifically

Reply:
yeah, learning what is ours and what isn't :D